.github/workflows/validate.yaml

```diff
@@ -55,7 +55,7 @@ jobs:
         if: (github.event_name == 'pull_request' && github.base_ref != 'main') || ((github.event_name == 'push' || github.event_name == 'workflow_dispatch') && github.ref_name != 'main')
         run: |
           tag="release-$(yq read ./olm-catalog/serverless-operator/project.yaml 'project.version')"
-          echo "CURRENT_VERSION_IMAGES=${tag}" >> $GITHUB_ENV
+          yq write ./olm-catalog/serverless-operator/project.yaml 'project.tag' ${tag}
```

Comment:
Should we do this write in `bumpso` ? https://github.com/openshift-knative/serverless-operator/blob/main/hack/cmd/bumpso/bumpso.go

Reply:
I don't think so. For main branch we want this to be always `project.tag: main`. But for release branches we want it to be specific, e.g. `release-1.34.0`.
AFAIK, we run the `bumpso` github action against main: https://github.com/openshift-knative/serverless-operator/blob/main/.github/workflows/bump.yaml#L29
Not sure if I'm missing something?

Reply:
OK. Looks like this is completely replaced by https://github.com/openshift-knative/serverless-operator/pull/2827 .
The version of index image will be determined from project version, e.g. for 1.35.0 it will be `docker://quay.io/redhat-user-workloads/ocp-serverless-tenant/serverless-operator-135/serverless-index:latest` 
Closing.